tests/DataDistributionMetrics.txt

```diff
@@ -0,0 +1,4 @@
+testTitle=DataDistributionMetrics
+testName=DataDistributionMetrics
+numTransactions=100
+writesPerTransaction=1000
```

Comment:
The test can be combined with failure events. For example, in `MoveKeysCycle.txt` test, it has
`testName=Cycle
    transactionsPerSecond=2500.0
    testDuration=10.0
    expectedRate=0.025

    testName=Attrition
    machinesToKill=1
    machinesToLeave=3
    reboot=true
    testDuration=10.0

    testName=Attrition
    machinesToKill=1
    machinesToLeave=3
    reboot=true
    testDuration=10.0`

Adding the attrition will make sure your feature is robust in machine failure.
Adding CycleTest makes sure your feature won't affect the existing system's correctness.